src/main/resources/tanagra/cms_synpuf/expanded/cms_synpuf.json

```diff
@@ -3,19 +3,19 @@
   "dataPointers" : [ {
     "type" : "BQ_DATASET",
     "name" : "index_dataset",
-    "projectId" : "broad-tanagra-dev",
-    "datasetId" : "cmssynpuf_index_011523",
-    "queryProjectId" : "broad-tanagra-dev"
+    "projectId" : "broad-jade-ps-data",
+    "datasetId" : "cmssynpuf_index_1",
+    "queryProjectId" : "broad-jade-ps-data"
   }, {
-    "type" : "BQ_DATASET",
+    "type" : "AZURE_DATASET",
     "name" : "omop_dataset",
-    "projectId" : "bigquery-public-data",
-    "datasetId" : "cms_synthetic_patient_data_omop",
-    "queryProjectId" : "broad-tanagra-dev"
+    "datasetId" : "ba420228-f13a-490a-93fc-fffd91e0f51b",
+    "datasetName" : "synthetic_omop_data_20_participants_v2",
+    "userName" : "pshapiro@test.firecloud.org"
```

Comment:
I know you mentioned that this is used to create the signed URL. Is there any risk of other people not being able to use this with your username here? If so, should we put a placeholder here and add replacing it with your own azure account to the index runner doc?

Reply:
That's a good idea, I can do that.